sdk/src/pubkey.rs

```diff
@@ -248,35 +248,35 @@ mod tests {
         let program_id = Pubkey::from_str("BPFLoader1111111111111111111111111111111111").unwrap();
 
         assert_eq!(
-            Pubkey::create_program_address(&[exceeded_seed], &program_id),
+            Pubkey::create_program_address(&[exceeded_seed.as_ref()], &program_id),
```

Comment:
`exceeded_seed` and `max_seed` don't have to be utf-8 anymore, maybe change their declaration?